PICMI_Python/base.py

```diff
@@ -26,8 +32,37 @@ def register_constants(implementation_constants):
 def _get_constants():
     return _implementation_constants
 
+
+VectorFloat3 = typing.NewType('VectorFloat3', typing.Union[Sequence[float], np.ndarray[3, np.float64]])
+VectorInt3 = typing.NewType('VectorInt3', typing.Union[Sequence[int], np.ndarray[3, np.int64]])
+Expression = typing.NewType('Expression', str)
```

Comment:
How do you fell about moving this into a separate *namespace* (for python a module i.e. here a file)?

This way every type will get a prefix and the distinction between python native & PICMI custom (and perhaps other types) will become clear.

Reply:
had that problem too, [PEP 484 allows using strings instead of literals for forward declarations](https://peps.python.org/pep-0484/#the-problem-of-forward-declarations)

But ultimately your call.

Reply:
Ok, thanks, using strings worked. I am clearly not the first person to come across this circular definition problem :)